### PR TITLE
Hide binding details if the svc instance is not bindable

### LINKED ISSUE #456: Non-bindable ClusterServiceClass
I am not able to bind a Mysql instance class in the basic plan, is it possible that the UI let me choose it but it should have not? Is this something that we can detect?

selection_629

### PR DESCRIPTION
**Pull request by andresmgot**
Fixes #456 

screenshot 2018-11-09 at 09 59 16

### CODE REVIEW

File: dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx [hunk 1]
```diff
@@ -162,15 +162,44 @@ class ServiceInstanceView extends React.Component<IServiceInstanceViewProps> {
...
+            )}
+            <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
+          </div>
         );
-      bindingList = (
-        <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
-      );
+      } else {
+        bindingSection = (
+          <h5 className="padding-b-big">
```
Comment (prydonius) [inline dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx]: I would keep the `<h2>Bindings</h2>` heading, and then just add a paragraph with a message like "This Instance cannot be bound to applications."

File: dashboard/src/components/ServiceInstanceView/ServiceInstanceView.test.tsx [hunk 2]
```diff
@@ -208,5 +230,32 @@ context("when all the components are loaded", () => {
       ).toExist();
       expect(wrapper).toMatchSnapshot();
     });
+
+    it("should not show bindings information if the class is not bindable", () => {
+      const classes = {
```
Comment (migmartri) [inline dashboard/src/components/ServiceInstanceView/ServiceInstanceView.test.tsx]: Maybe you can share these `classes` in the tests and only override the `spec.bindable` value during mount?

**Comment by andresmgot** [comment]
@prydonius PTAL, do you think the format is okay now?

File: dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx [hunk 3]
```diff
@@ -162,15 +162,39 @@ class ServiceInstanceView extends React.Component<IServiceInstanceViewProps> {
...
+              />
+            )}
+            <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
+          </div>
         );
-      bindingList = (
-        <BindingList bindingsWithSecrets={bindings} removeBinding={this.props.removeBinding} />
-      );
+      } else {
+        bindingSection = <p>This instance cannot be bound to applications</p>;
```
Comment (prydonius) [inline dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx]: let's add a full stop at the end of this sentence

**Pull request merged**